content/hardware/03.nano/boards/nano-matter/tutorials/matter-fan/content.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Matter Smart Fan'
+difficulty: beginner
+compatible-products: [nano-matter]
+description: 'Learn how to build a Matter fan with speed control.'
+tags:
+  - IoT
+  - Matter
+  - BLE
+  - Fan
+  - Motor
+author: 'Christopher Méndez'
+hardware:
+  - hardware/03.nano/boards/nano-matter
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+---
+
+## Overview
+
+This tutorial will teach you how to create a Matter fan for keeping you cool. 
+
+![Matter fan overview](assets/thumbnail-v4.png)
+
+Thanks to the seamless compatibility of the Nano Matter with almost any Matter network we can easily integrate our fan with Amazon Alexa, Google Assistant, Apple Home, Home Assistant and even custom assistants.
+
+We have prepared a short demo in video format in case you are a visual learner.
+
+<iframe width="100%" height="480" src="https://www.youtube.com/embed/AiFm4T9EfnA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Hardware and Software Requirements
+### Hardware Requirements
+
+- [Nano Matter](https://store.arduino.cc/products/nano-matter) (x1)
+- Grove motor driver module (x1)
+- DC motor (x1)
+- Breadboard (x1)
+- Jumper wires
+- Google Nest Hub Max (Thread Border Router) (x1)
+- [USB-C® cable](https://store.arduino.cc/products/usb-cable2in1-type-c) (x1)
+
+### Software Requirements
+
+- [Arduino IDE 2.0+](https://www.arduino.cc/en/software) or [Arduino Cloud Editor](https://create.arduino.cc/editor)
+- [Google Home App](https://home.google.com/get-app/)
+
+### Download the Project Code
+
+[![ ](assets/download.png)](assets/matter_fan.zip)
+
+Download the complete project code [here](assets/matter_fan.zip).
+
+### Board Core and Libraries
+
+The **Silicon Labs** core contains the libraries and examples you need to work with the board's components, such as its Matter, Bluetooth® Low Energy, and I/Os. To install the Nano Matter core, navigate to **File > Preferences** and in the **Additional boards manager URLs**, add the following:
+
+`https://siliconlabs.github.io/arduino/package_arduinosilabs_index.json`
+
+Now navigate to **Tools > Board > Boards Manager** or click the Boards Manager icon in the left tab of the IDE. In the Boards Manager tab, search for `Nano Matter` and install the latest `Silicon Labs` core version.
+
+![Installing the Silicon Labs core in the Arduino IDE](assets/bsp-install-2.png)
+
+## Project Setup
+
+### Schematic Diagram
+
+Use the following connection diagram for the project:
+
+![Project wiring diagram](assets/diagram-v4.png)
+
```

Comment:
add two lines explanation about the general connection or **the main thing the user must take attention**